web/meetup-facilitator/src/pages/Preferences.jsx

```diff
@@ -59,7 +59,7 @@ function Preferences() {
         // TODO: make post request 
 
 
-        navigate('/');
+        navigate('/recommendation');
```

Comment:
Then this can be changed to ``/recommendation/${group_id}``

Reply:
Additionally, let's make it plural `recommendations` to be consistent with the other pages.